torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5352,6 +5352,17 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                # https://github.com/pytorch/pytorch/pull/57934#issuecomment-840091579
                SkipInfo('TestOpInfo', 'test_unsupported_dtypes',
                         device_type='cuda', dtypes=(torch.bfloat16,)),)),
+    OpInfo('__rmod__',
+           op=torch.Tensor.__rmod__,
+           dtypes=all_types_and(torch.bfloat16, torch.half),
+           dtypesIfCPU=floating_types_and(torch.half,),
+           dtypesIfCUDA=all_types_and(torch.bfloat16, torch.half, torch.bool),
+           sample_inputs_func=sample_inputs_rbinops,
+           supports_out=False,
+           skips=(SkipInfo('TestCommon', 'test_variant_consistency_jit',),),
+           supports_autograd=False,
```

Comment:
Interesting... why doesn't this support autograd? torch.remainder supports autograd typically, right?

Reply:
`torch.remainder` seems not to support autograd of the second argument currently.
```py
>>> x1 = torch.tensor([3.], requires_grad=True)
>>> x2 = torch.tensor([5.], requires_grad=True)
>>> y = x1 % x2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: the derivative for 'other' is not implemented
```

Reply:
Interesting. Can we mark the operation as supporting autograd but, when `requires_grad=True`, mark the second input as not requiring gradient still? We can add a comment explaining why.

Or is there some reason why that wouldn't work?